notification-portlet-webcomponents/notification-icon/src/components/ieDropdown.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="dropdown">
+    <button class="btn btn-secondary dropdown-toggle"
+            type="button"
+            id="dropdownMenuButton"
+            data-toggle="dropdown"
+            aria-haspopup="true"
+            aria-expanded="false"
```

Comment:
heya @mindblender 👋 , just a small comment about aria. you need to provide a way to set this attribute `aria-expanded` to `true`, when the button is activated. And toggle its state to `false`, when dropdown is hidden.